README.md

```diff
@@ -103,7 +104,11 @@
 
 ### Deployment
 
-**This project is deployed by the author, no permission for deployment by any other client.**
+- Find the link to the deployed version here [Expense Controller](https://expense-controller-app.onrender.com/)
+
+### Presentation video
+
+- Find the link to the loom presentation video here [video](https://www.loom.com/share/317d06bee0724461b76ad07703a441f8)
```

Comment:
- [ ] Please make sure the maximum length of the presentation video is 5 minutes long and in the video kindly highlight some interesting piece of code as requested in the project requirements. Currently, the video you submitted is 7+ minutes long and it doesn't include highlighted code.